plugins/inputs/win_services/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Plugin: win_services
+Input plugin to report Windows services info: service name, display name, state, startup mode
+
+It requires that Telegraf must be running under the administrator privileges.
+### Configuration:
+
+```toml
+[[inputs.win_services]]
+  ## Names of the services to monitor. Leave empty to monitor all the available services on the host
+  service_names = [
+    "LanmanServer",
+    "TermService",
+  ]
+```
+
+### Measurements & Fields:
+
+- win_services
+    - state
+    - startup_mode
+
+The `state` tag can have the following values:
+* _stopped_         
+* _start_pending_   
+* _stop_pending_    
+* _running_         
+* _continue_pending_
+* _pause_pending_   
+* _paused_
+
+The `startup_mode` tag can have the following values:
+* _boot_start_  
+* _system_start_
+* _auto_start_  
+* _demand_start_
+* _disabled_
+
+### Tags:
+
+- All measurements have the following tags:
+    - service_name
+    - display_name
+
+### Example Output:
+
+Using the default configuration:
+
+When run with:
+```
+E:\Telegraf>telegraf.exe -config telegraf.conf -test
+```
+It produces:
+```
+* Plugin: inputs.win_services, Collection 1
```

Comment:
You can remove everything in this section up through this line, so the only items are the example line protocol